go.mod

```diff
@@ -48,6 +49,7 @@ require (
 	github.com/cosmos/btcutil v1.0.4 // indirect
 	github.com/cosmos/gorocksdb v1.2.0 // indirect
 	github.com/cosmos/iavl v0.17.3 // indirect
+	github.com/cosmos/ibc-go v1.2.3 // indirect
```

Comment:
This is imported by Desmos v2, which shouldn't be there. Make sure you delete all the occurrences using this dependency and then run `go mod tidy` to clean up the `go.mod` file